src/tools/tablemeta_reader.cc

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2021 4Paradigm
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "tools/tablemeta_reader.h"
+
+#include <stdlib.h>
+
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "base/glog_wapper.h"
+#include "codec/schema_codec.h"
+#include "proto/name_server.pb.h"
+
+using ::openmldb::codec::SchemaCodec;
+
+namespace openmldb {
+namespace tools {
+
+void TablemetaReader::CopyFromRemote(const std::string& host, const std::string& source,
+                                     const std::string& dest, int type) {
+    char exec[200];
```

Comment:
- add `user_name` gflag to specify users in `SCP` command. the default value is empty string.
- update README at same time